Rock/Model/CodeGenerated/DataViewService.cs

```diff
@@ -83,6 +84,55 @@ public bool CanDelete( DataView item, out string errorMessage )
             }  
             return true;
         }
+
+        /// <summary>
+        /// Create a new non-persisted Data View using an existing Data View as a template. 
+        /// </summary>
+        /// <param name="dataViewId">The identifier of a Data View to use as a template for the new Data View.</param>
+        /// <returns></returns>
+        public DataView GetNewFromTemplate(int dataViewId)
+        {
+            var item = this.Queryable()
+                           .AsNoTracking()
+                           .Include( x => x.DataViewFilter )
+                           .FirstOrDefault( x => x.Id == dataViewId );
+
+            if (item == null)
```

Comment:
Yeah, I missed that one - sorry. I know the standard, it just doesn't come naturally to me.